modules/acquia_cms_site_studio/acquia_cms_site_studio.install

```diff
@@ -120,6 +122,24 @@ function acquia_cms_site_studio_install($is_syncing) {
     $config_storage = \Drupal::service('config.storage');
     $config_storage->write('editor.editor.cohesion', $source_optional_dir->read('editor.editor.cohesion'));
     $config_storage->write('filter.format.cohesion', $source_optional_dir->read('filter.format.cohesion'));
+
+    // Re-write filter format filtered html.
+    $editorObject = $entity_type_manager
+      ->getStorage('editor')
+      ->load('filtered_html');
+    if ($editorObject instanceof Editor) {
+      $filtersObj = $editorObject->getFilterFormat();
+      $filters_instance_ids = $editorObject->getFilterFormat()->filters()->getInstanceIds();
```

Comment:
@chandan-singh7929 replace **$editorObject->getFilterFormat()** with $filtersObj.
As discussed we have to follow same pattern throughout the codebase. 
.install .module file variables naming convention are in snake_case
classes plugins services should be in camelCase. Please make changes accordingly.

Reply:
@apathak18 - I have made the changes, thank you!